hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OmRPCLoadGenerator.java

```diff
@@ -0,0 +1,123 @@
+package org.apache.hadoop.ozone.freon;
```

Comment:
License header is needed in all files.  Example:

https://github.com/apache/ozone/blob/050d58b6d0f59e251519a4108930fa78f850b231/hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/scm/net/TestNetUtils.java#L1-L17

Reply:
Thank you Attila! sure, added!